### PR TITLE
(#1) Show weekday next to date

### PR DESCRIPTION
**Pull request by vtno**
This PR just add 2 functions that:
1. Get the weekday
2. Format the date

Might be good to have some suggestion on a good date format as this is basically just weekday + date

e.g. `Tuesday 2021-02-23`

EDITED

The final format as discussed is `2021-02-23 (Tuesday)` and I've also added a check to exit with error when `data` project does not exist.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,21 @@
 # Clone data repository
 git clone https://github.com/clousehub/thaiclubhouse-data.git data
 
+```
+# Install PHP CLI and composeer
+
+**Ubuntu**
+``` sh
+sudo apt install -y php-7.4 php7.4-mbstring # required for json5 lib
+sudo curl -sS https://getcomposer.org/installer | php
```
Comment (dtinth) [inline README.md]: Can install composer with apt as well 😄 

```suggestion
sudo apt install -y php-7.4 php7.4-mbstring composer
```

**Comment by vtno** [comment]
@dtinth This is the error I was talking about

```sh
PHP Notice:  Undefined index: events in /Users/tino.thamjarat/Workspace/external/tino/thaiclubhouse/gen.php on line 21

Notice: Undefined index: events in /Users/tino.thamjarat/Workspace/external/tino/thaiclubhouse/gen.php on line 21
PHP Warning:  Invalid argument supplied for foreach() in /Users/tino.thamjarat/Workspace/external/tino/thaiclubhouse/gen.php on line 21

Warning: Invalid argument supplied for foreach() in /Users/tino.thamjarat/Workspace/external/tino/thaiclubhouse/gen.php on line 21
2021-02-28
PHP Notice:  Undefined index: events in /Users/tino.thamjarat/Workspace/external/tino/thaiclubhouse/gen.php on line 93
PHP Warning:  array_filter() expects parameter 1 to be array, null given in /Users/tino.thamjarat/Workspace/external/tino/thaiclubhouse/gen.php on line 93
PHP Warning:  uksort() expects parameter 1 to be array, null given in /Users/tino.thamjarat/Workspace/external/tino/thaiclubhouse/gen.php on line 96
PHP Warning:  Invalid argument supplied for foreach() in /Users/tino.thamjarat/Workspace/external/tino/thaiclubhouse/gen.php on line 105
public/index.html
```

**Review by dtinth — APPROVED** [review]
This looks advanced, didn’t expect it to come with unit tests!

I learn a lot about PHP today because I never used PHPUnit and how composer can autoload files from `src` folder 😂

**Comment by vtno** [comment]
@dtinth I just learn those today because I want to write test as well!

**Pull request merged**